### PR TITLE
Switch standardnotes to x64

### LINKED ISSUE #7469: Standard Notes 3.9.5 update fails
## Bug Report

**Package Name:** standardnotes

### Current Behaviour

404 error when updating to 3.9.5

```powershell
Updating 'standardnotes' (3.8.21 -> 3.9.5)
Downloading new version
OperationStopped: C:\Users\greg\scoop\apps\scoop\current\lib\install.ps1:130
Line |
 130 |          throw $e
     |          ~~~~~~~~
     | The remote server returned an error: (404) Not Found.
```

### Expected Behaviour

Application updates successfully

### Possible Solution

URL is missing -x64. 
Current -> `https://github.com/standardnotes/desktop/releases/download/v3.9.5/standard-notes-3.9.5-win.exe`
Actual -> `https://github.com/standardnotes/desktop/releases/download/v3.9.5/standard-notes-3.9.5-win-x64.exe`

I am also getting a different hash for the download. 
```powershell
> Get-FileHash -Path "standard-notes-3.9.5-win-x64.exe" -Algorithm SHA512 | fl

Algorithm : SHA512
Hash      : ACD38CEC621CBB7BE70C3DC6653BE7CF362B4A5F4C7B7C3B71F0593E700A506C3B3CB74D63EAF15A7E57C303E16ACC0FC061B41A4CAAE2443B962900A0D56446
```

### System details

**Windows version:** [e.g. 7, 8, 10]
10
**OS architecture:** [e.g. 32bit, 64bit]
64bit
**PowerShell version:** [output of `"$($PSVersionTable.PSVersion)"`]
7.2

### PR DESCRIPTION
**Pull request by gregbeck**
* Change the auto update URL to grab the x64 file
* Change the hash url to use the SHA256SUMS file which the default parser can handle.
* Use checkver.ps1 to update the URL and hash using the above changes

Closes #7469

### CODE REVIEW

**Comment by rashil2000** [comment]
Since the file is now 64bit, you'll need to use the architecture block, for URL, hash, and autoupdate, see https://github.com/ScoopInstaller/Extras/blob/master/bucket/neovide.json

**Comment by gregbeck** [comment]
The copy in the repo was updated to 3.9.7.  I merged in the upstream changes, then added the architecture blocks and updated the paths to 3.9.7.

**Comment by rashil2000** [comment]
Does the binary "Standard Notes.exe" accept any commandline arguments?

**Comment by gregbeck** [comment]
I wasn't able to find any command line args in the Standard Notes docs.

**Comment by rashil2000** [comment]
Then you can remove the `bin` field.

**Comment by gregbeck** [comment]
Done

File: bucket/standardnotes.json [hunk 1]
```diff
@@ -25,10 +21,13 @@
...
     "autoupdate": {
-        "url": "https://github.com/standardnotes/desktop/releases/download/v$version/standard-notes-$version-win.exe#/dl.7z",
-        "hash": {
-            "url": "$baseurl/latest.yml",
-            "regex": "sha512:\\s+$base64"
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/standardnotes/desktop/releases/download/v$version/standard-notes-$version-win-x64.exe#/dl.7z",
+                "hash": {
+                    "url": "$baseurl/SHA256SUMS"
```
Comment (niheaven) [inline bucket/standardnotes.json:28]: Why you use sha256 when there is sha512? Does original au.hash break?

Comment (gregbeck) [reply bucket/standardnotes.json:28]: The original configuration is pulling the sha512 from a [yaml file](https://github.com/standardnotes/desktop/releases/download/v3.9.7/latest.yml).  The RegEx is capturing the sha512 for the non-x64 exe file which the Standard Notes project doesn't seem to be posting anymore. I didn't see a yaml parser in the autoupdate documents so I figured the default parser for the SHA256SUMS file was less likely to break.  It seemed like I would need a complex RegEx to parse the yaml file.

Comment (cderv) [reply bucket/standardnotes.json:28]: As I was also trying to fix this, I think you can use this regex for the YAML file to keep the same workflow (looking in latest.yml)

````json
    "autoupdate": {
        "url": "https://github.com/standardnotes/desktop/releases/download/v$version/standard-notes-$version-win-x64.exe#/dl.7z",
        "hash": {
            "url": "$baseurl/latest.yml",
            "regex": "\\s+- url: $basename\\s+sha512:\\s+$base64"
        }
    }
````

This works on my side to get the correct hash.

**Review by niheaven — CHANGES_REQUESTED** [review]
Sorry my review is not comment...

**Comment by gregbeck** [comment]
I seem to got myself stuck with changes I did not intend in this PR and haven't figured out how to clean it up.  Would it be ok if I closed this PR and created a new one with just the intended changes?

**Comment by rashil2000** [comment]
Yes, please create a new PR

**Comment by gregbeck** [comment]
Closing.  Will create a new PR

**Pull request closed without merging**